cosi/threshold/types/mod.go

```diff
@@ -265,10 +279,16 @@ func NewThresholdVerifierFactory(fac crypto.VerifierFactory) crypto.VerifierFact
 	}
 }
 
+// FromAuthority implements crypto.VerifierFactory. It creates a verifier from
+// the authority so that the mask's signature will pick the participants that
+// have participated. The ordering of the authority must be the same.
```

Comment:
Newbie question:
`The ordering of the authority must be the same.`
what does this mean?

Reply:
The signature mask defines how participated from an ordered list of signers. If the list is shuffled, the aggregate public key will be wrong. The authority abstraction is there to prevent this, but you could still use a different one with a different order.